lmdk/include/adsp_stddef.h

```diff
@@ -0,0 +1,108 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2021 Intel Corporation. All rights reserved.
+
+
+#ifndef _ADSP_STDDEF_H_
+#define _ADSP_STDDEF_H_
```

Comment:
@RanderWang Why do we need to copy headers? It seems in this example copying headers is not needed -> https://github.com/thesofproject/sof/pull/8546

I thought now that we have headers in separate module folder, we no longer needed this. And 8546 seems to show this is now needed.

Something I'm missing?

Reply:
o, I did part time work on loadable smart amp long time ago and am out of date. Let me check the latest status